### PR TITLE
ci: enable pytest coverage and unit tests in CI

### PR DESCRIPTION
**Pull request by Pro**
See also
https://github.com/marketplace/actions/pytest-coverage-comment

See also https://github.com/devopshq/artifactory-cleanup/pull/48#issuecomment-1172298856

### CODE REVIEW

File: .github/workflows/pytest-coverage.yml [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# This workflow will install dependencies, create coverage tests and run Pytest Coverage Comment
+# For more information see: https://github.com/MishaKav/pytest-coverage-comment/
+name: pytest-coverage-comment
```
Comment (allburov) [inline .github/workflows/pytest-coverage.yml:3]: Could we use https://github.com/devopshq/artifactory-cleanup/blob/16254bb8848fcff47d4d9b35c0bd7a4ef01b053e/.github/workflows/release.yml and run pytest after ` Install dependencies` step? 
If `release` name confuses, let's rename it to `build.yaml` to show that it's not only about releases

It'd be great to run tests before release!

File: .github/workflows/pytest-coverage.yml [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+      - name: Set up Python 3.8
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.8
+
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install flake8 pytest pytest-cov
```
Comment (allburov) [inline .github/workflows/pytest-coverage.yml:25]: It's better to pin library's versions in `requirements-dev.txt` file and install them from it.

Btw, why is there`flake8` module?

File: .github/workflows/pytest-coverage.yml (hunk 1 above)
Comment (Pro) [reply .github/workflows/pytest-coverage.yml]: Updated the workflow. Coverage is now not included anymore.

For this good enough. Long-term you should think about adding coverage (not in this MR)

**Comment by Pro** [comment]
@allburov this now seems to be ready. Feel free to squash merge

**Comment by allburov** [comment]
The repository is safe now. Thank you, @Pro!

**Pull request merged**